documentation/modules/security/proc-accessing-kafka-using-nodeports.adoc

```diff
@@ -100,8 +100,8 @@ status:
           -----BEGIN CERTIFICATE-----
           
           -----END CERTIFICATE-----
-      name: external
-      type: external
+      name: external4
+      type: external4
```

Comment:
```suggestion
      type: nodeport
```

Reply:
Maybe we should stop using it. It has been a long time since we deprecated it.

Reply:
Right, this is so confusing that I always do the same mistake.

+1 on stop using it.